modules/ROOT/pages/mule-concepts.adoc

```diff
@@ -73,11 +73,11 @@ Mule *transformers* are the key to exchanging data between nodes, as they allow
 
 Mule uses *components* to conduct backend processes for specific business logic such as checking customer and inventory databases. Components route messages to the correct application, such as an order fulfillment system. Mule uses http://en.wikipedia.org/wiki/Staged_event-driven_architecture[Staged Event-Driven Architecture (SEDA)] for core asynchronous message processing in flows. Importantly, components don't have to have any Mule-specific code; they can simply be POJOs, Spring beans, Java beans, Groovy scripts, or web services containing the business logic for processing data. Components can even be developed in other languages such as Python, JavaScript, Ruby, and PHP. Mule’s catalog of building blocks support the most commonly used Enterprise Integration Patterns.
 
-Flows and batch jobs can also include *filters*, *scopes*, and *routers*. For example, you can use a filter to whitelist IP addresses from which your application accepts messages; you can use a scope to "wrap" around several message processors and cache the result of the processing they perform; you can use a router to send messages down different paths in your application depending on the content of the message payload. Mule includes a variety of filters, scopes, and routers to customize how a flow or batch job processes messages.
+Flows and batch jobs can also include *filters*, *scopes*, and *routers*. For example, you can use a filter to allow IP addresses from which your application accepts messages; you can use a scope to "wrap" around several message processors and cache the result of the processing they perform; you can use a router to send messages down different paths in your application depending on the content of the message payload. Mule includes a variety of filters, scopes, and routers to customize how a flow or batch job processes messages.
```

Comment:
Per our style ref., use italics SPARINGLY for emphasis, rather than boldface:
```suggestion
Flows and batch jobs can also include _filters_, _scopes_, and _routers_. For example, you can use a filter to allow IP addresses from which your application accepts messages; you can use a scope to "wrap" around several message processors and cache the result of the processing they perform; you can use a router to send messages down different paths in your application depending on the content of the message payload. Mule includes a variety of filters, scopes, and routers to customize how a flow or batch job processes messages.
```
Consider making the narrative list a bulleted one, too.

Reply:
OTHERWISE...all you asked for was the "allow" thing, which is fine and I am going to go approve them all right now.